src/libs/actions/Report.js

```diff
@@ -133,12 +133,18 @@ function getParticipantEmailsFromReport({sharedReportList}) {
 }
 
 /**
- * Returns a generated report title based on the participants
+ * Returns the title for a default room or generates one based on the participants
  *
- * @param {Array} sharedReportList
+ * @param {Object} fullReport
+ * @param {String} chatType
  * @return {String}
  */
-function getChatReportName(sharedReportList) {
+function getChatReportName(fullReport, chatType) {
+    if (isDefaultRoom({chatType})) {
```

Comment:
Why couldn't you pass the report object here?

Reply:
The problem is we haven't created the "simplified" report object here yet. We're actually creating that simplified report with this `getChatReportName()`. I'd either do this or get it from `'reportNameValuePairs', 'chatType'` of the `report` object. That's why I called it "fullReport", but I think that's an unnecessary distinction now.